src/classNames.js

```diff
@@ -0,0 +1,15 @@
+import style from 'flexboxgrid';
+
+function defaultClassNameLookup(className) {
+  return style[className] || className;
```

Comment:
Hm, when I test webpack's `css-loader` without module support, `import style from 'flexboxgrid';` sets `style` to an empty object.

However, you're correct that's not really documented behavior and could be different across  implementations.   I've reworked.